### ISSUE TITLE
No forced SSL

### ISSUE THREAD

**Issue by Yakuza112** [vergecurrency/vergecurrency.com#426]
I may access vergecurrency.com via HTTP, as we use Cloudflare as CDN:

https://blog.cloudflare.com/how-to-make-your-site-https-only/

**Comment by jrsikors** [comment]
How are you accessing it via http?  When I try, I get re-directed to https.

**Comment by justinvforvendetta** [comment]
ill double check the cloudflare setup, but im pretty sure it's already forcing ssl. i know it did on the last website (the one based in jekyll)

**Comment by Yakuza112** [comment]
its not that big thing... as if i try to access a page via HTTP it redirects me to HTTPS

but, i am able to access files via HTTP, so, in my opinion its not strict https
-> for example: http://vergecurrency.com/static/img/press/logo/verge-logo-white.png

**Comment by jrsikors** [comment]
Yeah that's why I was interested in how you were accessing it via HTTP.  That might make a difference in how Justin approaches the issue.

So you are able to access files via http , but any web page will redirect you to https..
IE:
going to this PAGE:
http://vergecurrency.com/wallets/
will redirect you to https:
https://vergecurrency.com/wallets/

going to this FILE:
http://vergecurrency.com/static/img/press/logo/verge-logo-white.png

will not redirect https
